capstone/src/components/User/Toast.jsx

```diff
@@ -0,0 +1,19 @@
+// Show toast value
+export default function callToast({ toast, title, description, status }) {
```

Comment:
Personally I think it's a bit confusing to pass in the toast object. I would recommend something like:

```
function getToastOptions({title, description, status}) {
  if (status === 'error') {
    // return other toast options
    return {...}
  }
  return {
    title,
    description,
  // You can pass in defaults here
    status: status ?? 'success',
  }
}
```

```
const toast = useToast();
toast(getToastOptions({ /* options */ }));
```

Reply:
That actually makes so much sense! I just included that change, thanks for that!